index.html

```diff
@@ -1,534 +1,1010 @@
+          designer="Joe"
+          data-version-number="3.0"
+        />
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link" href="#despre">Despre</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#servicii">Servicii</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__link--btn btn--show-modal" href="#contact"
+              >Contact</a
+            >
+          </li>
+        </ul>
+        <button id="toggleButton-close">
+          <svg
+            class="btn-close__icon"
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            id="menu"
+          >
+            <path
+              d="M12,7a2,2,0,1,0-2-2A2,2,0,0,0,12,7Zm0,10a2,2,0,1,0,2,2A2,2,0,0,0,12,17Zm0-7a2,2,0,1,0,2,2A2,2,0,0,0,12,10Z"
+            ></path>
+          </svg>
+        </button>
+        <ul class="nav__links nav__links-m nav__links-m--close">
+          <li class="nav__item">
+            <a class="nav__link" href="#despre">Despre</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#servicii">Servicii</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link nav__link--btn btn--show-modal" href="#contact"
+              >Contact</a
+            >
+          </li>
+        </ul>
+      </nav>
 
-    <div class="header__title header__title-m">
-      <div class="header-container-m">
-        <h1>
-          Sebastian Gabriel
-          <!-- Green highlight effect -->
-          <span class="highlight">Nagy</span>
-        </h1>
-        <h4>Avocat definitiv Cluj-Napoca</h4>
-        <p>
-          <span class="header__desc">
-            Activitatea noastră este caracterizata de inovare și adaptabilitate la nevoile clientului, transparență,
-            atenție la detalii, experiență și expertiza de înaltă calitate și nu în ultimul rand, disponibilitate în
-            relația cu clienții.
-          </span>
-        </p>
+      <div class="header__title">
+        <div class="header-container">
+          <h1>
+            Sebastian Gabriel
+            <!-- Green highlight effect -->
+            <span class="highlight">Nagy</span>
+          </h1>
+          <h4>Avocat definitiv Cluj-Napoca</h4>
+          <p>
+            <span class="header__desc">
+              Activitatea noastră este caracterizata de inovare și
+              adaptabilitate la nevoile clientului, transparență, atenție la
+              detalii, experiență și expertiza de înaltă calitate și nu în
+              ultimul rand, disponibilitate în relația cu clienții.
+            </span>
+          </p>
+        </div>
         <!-- <button class="btn--text btn--scroll-to">Mai mult &DownArrow;</button> -->
-        <a class="btn--show-modal" href="#contact">Contactează</a>
+        <button class="btn--show-modal btn--text">Contactează</button>
```

Comment:
This button should scroll to the bottom of the website, not open the modal. Please put back the `href` attribute.